### ISSUE TITLE
[BUG] Server side depends on `equator` mod

### ISSUE THREAD

**Issue by JulianVennen** [DM-Earth/Cabricality#182]
### What Happened?

When you install a server from the mrpack it crashes because the mod equator is labelled as client-only but required by Cabricality.

### Steps to Reproduce

1. Download the MRPack
2. Create a server following the mrpack specification

### Cabricality Version Check

Yes, I'm playing with Cabricality 0.3.1

### Crash Log

https://mclo.gs/WcEgzP5

**Comment by JieningYu** [comment]
Try installing the mod `equator`.

**Comment by JulianVennen** [comment]
Yeah, that works, but it should not be labelled as unsupported on the server in the mrpack if it is required.

**Comment by KrLite** [comment]
**Equator** was not labeled client-only when **Cabricality** `0.3.1` released. However, is should indeed be a client-only rendering mod. In **Cabricality** `1.0` we will remove the dependencies of **Equator** on the server side. For now, please just install it manually.
